README.md

```diff
@@ -1,230 +1,42 @@
-
-These source code will provide some knowledge of how to use PDC. For more reference, one may check the documentation folder in this repository.
-
-# PDC on Cori
-
-If you are running `PDC` on Cori supercomputer, here are some tips you would need to follow:
-
-* On Cori, it is recommended to use `cc` as the default compiler when compiling PDC and its dependencies. 
-* When preparing compilation for `PDC` using `CMake`, it is suggested to append console argument `-DMPI_RUN_CMD=srun` so that `ctest` can be executed on Cori.
-* Sometimes, it might be helpful to unload `darshan` module before the installation. 
-
-* For opening an interactive job session on Cori, it is recommended to add `--gres=craynetwork:2` option to the `salloc` command:
-    ```bash
-    salloc -C haswell -N 4 -t 01:00:00 -q interactive --gres=craynetwork:2
-    ```
-* To launch the PDC server and the client, add `--gres=craynetwork:1` before the executables, for example:
-
-  * Run 4 server processes, each on one node in background:
-    ```bash
-    srun -N 4 -n  4 -c 2 --mem=25600 --cpu_bind=cores --gres=craynetwork:1 --overlap ./bin/pdc_server.exe &
-    ```
-
-  * Run 64 client processes that concurrently create 1000 objects in total:
-    ```bash
-    srun -N 4 -n 64 -c 2 --mem=25600 --cpu_bind=cores --gres=craynetwork:1 --overlap ./bin/create_obj_scale -r 1000
-    ```
-
-
-
+![GitHub release tag(latest by date)](https://img.shields.io/github/v/tag/hpc-io/pdc)
+![Spack](https://img.shields.io/spack/v/pdc)
+
+## Proactive Data Containers (PDC)
+Proactive Data Containers (PDC) software provides an object-focused data management API, a runtime system with a set of scalable data object management services, and tools for managing data objects stored in the PDC system. The PDC API allows efficient and transparent data movement in complex memory and storage hierarchy. The PDC runtime system performs data movement asynchronously and provides scalable metadata operations to find and manipulate data objects. PDC revolutionizes how data is managed and accessed by using object-centric abstractions to represent data that moves in the high-performance computing (HPC) memory and storage subsystems. PDC manages extensive metadata to describe data objects to find desired data efficiently as well as to store information in the data objects.
+
+Full documentation of PDC with installation instructions, code examples for using PDC API, and research publications are available at [pdc.readthedocs.io](https://pdc.readthedocs.io)
+
+More information and publications on PDC is available at https://sdm.lbl.gov/pdc
+
+If you use PDC in your research, please use the following citations:
+
+```
+@misc{byna:2017:pdc,
+  title = {Proactive Data Containers (PDC) v0.1},
+  author = {Byna, Suren and Dong, Bin and Tang, Houjun and Koziol, Quincey and Mu, Jingqing and Soumagne, Jerome and Vishwanath, Venkat and Warren, Richard and Tessier, François and USDOE},
```

Comment:
Sorry for noticing this now. Can remove USDOE at the end.

Reply:
no problem, done!